src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './Tabs.scss';
+import classNames from 'classnames';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab:Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  return (
+    <ul className="tabs-container">
+      {tabs.map(tab => (
+        <li
+          key={tab.id}
+          className={classNames(
+            'tab-title',
+            { 'tab-title--active': tab.id === selectedTabId },
+          )}
+        >
+          <button
+            type="button"
+            className="tab-title__button"
+            onClick={() => onTabSelected(tab)}
```

Comment:
What if the tab you're clicking on is already selected? Add this condition to decide whether you should call this callback or not